participants/celestine_auburger.json

```diff
@@ -5,7 +5,7 @@
 		"placeFamilyNameFirst": false,
 		"hideFamilyNameOnWebsite": false
 	},
-	"githubAccountName": "irenama",
+	"githubAccountName": "aubcel",
```

Comment:
This is a correct change, I asked him to change the handle here 👍🏻

I didn't know if who would register (first), so I asked both of them to update the github name for me.